http/backup_service.go

```diff
@@ -117,26 +117,16 @@ func (h *BackupHandler) handleCreate(w http.ResponseWriter, r *http.Request) {
 
 	files = append(files, bolt.DefaultFilename)
 
-	credBackupPath := filepath.Join(internalBackupPath, DefaultTokenFile)
+	credsExist, err := h.backupCredentials(internalBackupPath)
 
-	credPath, err := defaultTokenPath()
-	if err != nil {
-		h.HandleHTTPError(ctx, err, w)
-		return
-	}
-	token, err := ioutil.ReadFile(credPath)
 	if err != nil {
 		h.HandleHTTPError(ctx, err, w)
-		return
 	}
 
-	if err := ioutil.WriteFile(credBackupPath, []byte(token), 0600); err != nil {
-		h.HandleHTTPError(ctx, err, w)
-		return
```

Comment:
Why was this return removed?

Reply:
This return will be covered by the (currently missing) one just above it.